mailpoet/views/layout.html

```diff
@@ -192,40 +191,52 @@
-    <% set helpscout_form_id = 'e93d0423-1fa6-4bbc-9df9-c174f823c35f' %>
-  <% endif %>
-
-  <script type="text/javascript">!function(e,t,n){function a(){var e=t.getElementsByTagName("script")[0],n=t.createElement("script");n.type="text/javascript",n.async=!0,n.src="https://beacon-v2.helpscout.net",e.parentNode.insertBefore(n,e)}if(e.Beacon=n=function(t,n,a){e.Beacon.readyQueue.push({method:t,options:n,data:a})},n.readyQueue=[],"complete"===t.readyState)return a();e.attachEvent?e.attachEvent("onload",a):e.addEventListener("load",a,!1)}(window,document,window.Beacon||function(){});</script>
-
-  <script type="text/javascript">
-    if(window['Beacon'] !== undefined && window.hide_mailpoet_beacon !== true) {
-      window.Beacon('init', '<%= helpscout_form_id %>');
-
-      // HelpScout Beacon: Configuration
-      window.Beacon("config", {
-        icon: 'message',
-        zIndex: 50000,
-        instructions: "<%= __('Want to give feedback to the MailPoet team? Contact us here. Please provide as much information as possible!') %>",
-        showContactFields: true
+  <script type="text/javascript">window.DocsBotAI=window.DocsBotAI||{},DocsBotAI.init=function(c){return new Promise(function(e,o){var t=document.createElement("script");t.type="text/javascript",t.async=!0,t.src="https://widget.docsbot.ai/chat.js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(t,n),t.addEventListener("load",function(){window.DocsBotAI.mount({id:c.id,supportCallback:c.supportCallback,identify:c.identify,options:c.options});var t;t=function(n){return new Promise(function(e){if(document.querySelector(n))return e(document.querySelector(n));var o=new MutationObserver(function(t){document.querySelector(n)&&(e(document.querySelector(n)),o.disconnect())});o.observe(document.body,{childList:!0,subtree:!0})})},t&&t("#docsbotai-root").then(e).catch(o)}),t.addEventListener("error",function(t){o(t.message)})})};</script>
+  <% if (mailpoet_has_premium_support) %>
+    <script type="text/javascript">
+      DocsBotAI.init({
+        id: "TqTdebbGjJeUjrmBIFjh/kzFE5FBebBJiSJ2Tm0nR",
+      })
+    </script>
+  <% else %>
+    <script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=7e09bac9-4dd5-4dd2-83b3-a4fa6d2278ec"> </script>
+    <script type="text/javascript">
+      // // on page load always close widget
+      zE('messenger', 'hide');
+      // prefilling email address on open
+      zE('messenger:on', 'open', function () {
+        console.log('Open');
+        setTimeout(() => {
+          const iframe = document.querySelectorAll('iframe[name="Messaging window"]')[0];
+          let input = null;
+          if (iframe) {
+            input = iframe.contentWindow.document.querySelectorAll('input[type="email"]')[0];
+          }
+          if (input) {
+            input.value = mailpoet_sender_email_address;
+          }
+        }, 1500); // the time is maybe higher but sometimes the initialization takes longer
       });
```

Comment:
This code should prefill the email address to the ZD widget, but when I start with the clean site (I cleaned local storage where ZD seems to store data) I start with `Happy Bot` then I have to click `Ask human` and then it asks for the name first (see screenshot
<img width="407" alt="Screenshot 2023-08-04 at 15 39 10" src="https://github.com/mailpoet/mailpoet/assets/1082140/02a2b384-96a8-4d06-8bfd-c9b5344f8166">
). So the code for entering the email will not do anything because I'm not able to click `Ask human` and then fill name within 1.5 seconds.

Also when I go from DocBot by clicking on "Contact Support" I get to "Happy Bot" so I need to get through two bots to get to human. Is is possible to skip the HappyBot?